### DISCUSSION TITLE
Does trivy support scan OCI artifacts?

### DISCUSSION THREAD

**Discussion by caozhuozi** [aquasecurity/trivy#8153] [Q&A]
### Question

as title.

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by DmitriyLewen** [comment]
Hello @caozhuozi 

Trivy supports OCI images - https://trivy.dev/latest/docs/target/container_image/#oci-layout
But you need to keep image into file and use `--input` flag.

Regards, Dmitriy
